common/culture/traditions/00_realm_traditions.txt

```diff
@@ -699,6 +713,11 @@ tradition_collective_lands = {
 		4 = farmland.dds
 	}
 	
+	can_pick = {
+		# Warcraft
+		NOT = { has_cultural_tradition = tradition_ruling_caste }
```

Comment:
Tested and it works. It also shows up in vanilla code, such as line 337 of _00_combat_traditions_ `tradition_warriors_of_the_dry`.

```	
can_pick = {
		culture_not_pacifistic_trigger = yes
		custom_description = {
			text = culture_in_dryland_desc
			any_culture_county = {
				any_county_province = {
					OR = {
						terrain = drylands
						terrain = desert
					}
				}
			}
		}
		NOT = { has_cultural_tradition = tradition_mubarizuns } #To prevent stacking after hybridization
	}
```

Reply:
Have you tested creating a new culture and trying to pick both `tradition_collective_lands` and `tradition_ruling_caste` at the same time?

Reply:
Adjusted in 37ac8c203